src/client/firefox.js

```diff
@@ -35,6 +35,10 @@ export async function onConnect(connection: any, actions: Object) {
     wasmBinarySource: supportsWasm
   });
 
+  threadClient._parent.listWorkers().then(workers =>
```

Comment:
@nyrosmith `threadClient._parent` can be undefined, you probably need to check if it exists before you call `listWorkers`